src/views/Registration/RegistrationReviewConfirm.vue

```diff
@@ -149,23 +149,11 @@ export default class RegistrationReviewConfirm extends Vue {
   get documentDeliveryAdditionalValue (): string {
     const orgPersonList = this.getAddPeopleAndRoleStep.orgPeople
 
-    if (this.isTypeSoleProp) {
-      const emails = orgPersonList
-        .filter(op => op.roles.some(role => role.roleType === RoleTypes.PROPRIETOR))
-        .map(op => op.officer?.email)
-        .filter(e => !!e)
-      return emails.join(',')
-    }
-
-    if (this.isTypePartnership) {
-      const emails = orgPersonList
-        .filter(op => op.roles.some(role => role.roleType === RoleTypes.PARTNER))
```

Comment:
I _think_ a "partner" is just a proprietor for a GP.

Or maybe we will need a new role type after all. I don't know. It's a future ticket.

Reply:
@vysakh-menon-aot Any thoughts ^^ ?